pioneer/packages/joy-utils/src/transport/tokenomics.ts

```diff
@@ -341,4 +454,346 @@ export default class TokenomicsTransport extends BaseTransport {
+            color: '#2196f3'
+          },
+          cellData: [
+            ['STAKE', `${formatBalance(stake).split('.000').join('')}`],
+            ['CURRENT VOTES', `${commitments.length}`],
+            ['CURRENT APPLICANTS', `${applicants.length}`],
+            ['POSITIONS', `${council_size.toNumber()}`],
+            ['TERM LENGTH', `${Math.trunc(termLength)} days`],
+            ['STAGE ENDS IN', `${Math.trunc(((stage.asType('Voting').toNumber() - currentBlockheight) * 6) / 3600)}h`]
+          ],
+          extra: {
+            termReward: statusData ? `$${(Number(statusData.price) * termReward).toFixed(2)}` : 'Not available..',
+            termMaxBonus: '$60',
+            text: "Both <a href='/#/council'>Council Members</a> and Voters can earn rewards:",
+            button: {
+              href: '/#/council/votes',
+              text: 'Vote Here'
+            }
+          }
+        }
+      ];
+    } else if (stage.isOfType('Revealing')) {
+      const commitments = (await this.api.query.councilElection.commitments()) as Vec<Hash>;
+      const votes = (await this.api.query.councilElection.votes.multi<SealedVote>(commitments));
+      const numberOfRevealedVotes = votes.reduce(
+        (revealedVoteCount, vote) => (vote.vote.unwrapOr(null) === null ? revealedVoteCount : revealedVoteCount + 1),
+        0
+      );
+      const applicants = (await this.api.query.councilElection.applicants()) as Vec<AccountId>;
+      const stake = votes.reduce((result, vote) => result + (vote.stake.transferred.toNumber() + vote.stake.new.toNumber()), 0);
+
+      return [
+        {
+          type: 'Council',
+          title: {
+            value: `Council election #${Number(round)} - Revealing stage`,
+            color: 'red'
+          },
+          cellData: [
+            ['STAKE', `${formatBalance(stake).split('.000').join('')}`],
+            ['REVEALED VOTES', `${numberOfRevealedVotes}/${votes.length}`],
+            ['CURRENT APPLICANTS', `${applicants.length}`],
+            ['POSITIONS', `${council_size.toNumber()}`],
+            ['TERM LENGTH', `${Math.trunc(termLength)} days`],
+            [
+              'STAGE ENDS IN',
+              `${Math.trunc(((stage.asType('Revealing').toNumber() - currentBlockheight) * 6) / 3600)}h`
+            ]
+          ],
+          extra: {
+            termReward: statusData ? `$${(Number(statusData.price) * termReward).toFixed(2)}` : 'Not available..',
+            termMaxBonus: '$60',
+            text: "If you've voted for a <a href='/#/council'>Council Member</a>:",
+            button: {
+              href: '/#/council',
+              text: 'Reveal Here'
+            }
+          }
+        }
+      ];
+    }
+
+    return undefined;
+  }
+
+  async openingData (group: 'storageWorkingGroup' | 'contentWorkingGroup'): Promise<LandingPageWorkingGroupExtra[]> {
+    let openingsToShow: Opening[];
+
+    if (group === 'storageWorkingGroup') {
```

Comment:
For Babylon and in the future you should use `workingGroupsT.queryByGroup` for any queries that can be executed in multiple working group contexts (this was probably not yet available since you based your work on `master`, but just pointing for once you get to upgrade this code)